sensing/autoware_pointcloud_preprocessor/src/blockage_diag/blockage_diag_node.cpp

```diff
@@ -190,6 +190,12 @@ void BlockageDiagComponent::filter(
     sky_blockage_range_deg_[1] = angle_range_deg_[1];
   } else {
     for (const auto p : pcl_input->points) {
+      if (p.channel >= vertical_bins) {
+        std::ostringstream oss;
```

Comment:
@vividf 
Q: Is there any reason to select `std::ostringstream` instead of `RCLCPP_ERROR`? I think `RCLCPP_ERROR` is common for autoware and this will keep error log in `ros/log` file?